javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java

```diff
@@ -415,19 +415,28 @@ && shouldGeneratePagingMethods()) {
                 } else if (operation.getExtensions() != null && operation.getExtensions().isXmsLongRunningOperation()
                         && (settings.isFluent() || settings.getPollingConfig("default") != null)
                         && !syncReturnType.equals(ClassType.InputStream)) {         // temporary skip InputStream, no idea how to do this in PollerFlux
+                    JavaVisibility simpleAsyncMethodVisibility =
+                            methodVisibility(ClientMethodType.SimpleAsyncRestResponse, false);
+                    JavaVisibility simpleAsyncMethodVisibilityWithContext =
+                            methodVisibility(ClientMethodType.SimpleAsyncRestResponse, true);
+                    if (settings.isDataPlaneClient()) {
+                        simpleAsyncMethodVisibility = JavaVisibility.Private;
+                        simpleAsyncMethodVisibilityWithContext = JavaVisibility.Private;
+                    }
+
                     // WithResponseAsync, with required and optional parameters
                     methods.add(builder
                             .returnValue(createSimpleAsyncRestResponseReturnValue(operation, proxyMethod, syncReturnType))
                             .name(proxyMethod.getSimpleAsyncRestResponseMethodName())
                             .onlyRequiredParameters(false)
                             .type(ClientMethodType.SimpleAsyncRestResponse)
                             .isGroupedParameterRequired(false)
-                            .methodVisibility(methodVisibility(ClientMethodType.SimpleAsyncRestResponse, false))
+                            .methodVisibility(simpleAsyncMethodVisibility)
                             .build());
 
                     if (settings.isContextClientMethodParameter()) {
                         addClientMethodWithContext(methods,
-                                builder.methodVisibility(methodVisibility(ClientMethodType.SimpleAsyncRestResponse, true)),
+                                builder.methodVisibility(simpleAsyncMethodVisibilityWithContext),
                                 parameters);
                     }
```

Comment:
Code change 1: If DPG and LRO, set client method with SimpleAsyncRestResponse to private.